aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie80/Models/MovieDateRatingDA.cs

```diff
@@ -17,7 +17,7 @@ public class Movie
     // </snippet11>
 
     // <snippet2>
-    // [Display(Name = "Release Date")]
+    [Display(Name = "Release Date")]
```

Comment:
```suggestion
```
My guess is that it should be removed. @wadepickett ?

Reply:
Or maybe not this late in the tutorial. I think it was used earlier so should be there. The added attributes need highlighting.

Reply:
Yes, it would be more appropriate to remove this part of the document.